csrc/kernel_cache.cpp

```diff
@@ -1130,9 +1148,11 @@ void FusionKernelRuntime::compileFusionParallel(KernelArgumentHolder args) {
 
     if (num_groups == 1 || isOptionDisabled(DisableOption::ParallelCompile)) {
       FUSER_PERF_SCOPE("FusionKernelRuntime::compileFusionParallel");
+      segment_profiler_start_compile(group_id, args.getDeviceIndex());
```

Comment:
Call this inside `compileKernel`? Similar to my previous comments, I think it's better to let compileKernel measure the time and directly put it in the SegmentProfiler. Up to you!

Reply:
If I start and stop inside `compileKernel`, then it will record sequential compile times in the parallel case.  The segment compile time is only recorded when the parallel compile is disabled.

Reply:
> The segment compile time is only recorded when the parallel compile is disabled.

Got it. If you haven't, I'd like to see that written down somewhere (e.g. in stdout) so users are not confused why they are not seeing the compilation time for each segment. 

That being said, I think, even with parallel compilation, there's value to measure each segment's compile time. For example, to identify long tails that make parallelization less useful than expected. Up to you!

Reply:
I think you are correct with this assessment as well.  I will still record the segment time so we can determine the long-pole which was the original intention in adding the column.